### PR TITLE
CNV 12997: Adding Known Issues for OCP upgrade errors on CNV 2.6.5

### PR DESCRIPTION
**Pull request by mgarrell777**
This PR addresses JIRA story [CNV-12997](https://issues.redhat.com/browse/CNV-12997) ( https://issues.redhat.com/browse/CNV-12997 ) which covers 3 Known Issue bugs:

https://bugzilla.redhat.com/show_bug.cgi?id=1979114
https://bugzilla.redhat.com/show_bug.cgi?id=1979116
https://bugzilla.redhat.com/show_bug.cgi?id=1977037

This story and these bugs pertain to various errors received when upgrading to OCP 4.8 when running CNV 2.6.5

CP: 4.8

Preview: At the very end of https://deploy-preview-34736--osdocs.netlify.app/openshift-enterprise/latest/virt/virt-4-8-release-notes?utm_source=github&utm_campaign=bot_dp#virt-4-8-known-issues.

### CODE REVIEW

File: virt/virt-4-8-release-notes.adoc [hunk 1]
```diff
@@ -214,3 +214,35 @@ No model registered for VirtualMachines
...
+** Creating a virtual machine from the Customize wizard provides a unclear and non-specific error message:
++
+[source]
+----
+Error creating virtual machine
+----
++
+*** As a workaround, copy the manifest and create the virtual machine from the CLI. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1979116[*BZ#1979116*])
++
+** When connecting to the VNC console by using the {VirtProductName} web console and `virtctl` command, sometimes the VNC console fails to respond. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1977037[*BZ#1977037*])
```
Comment (yaacov) [inline virt/virt-4-8-release-notes.adoc:248]: In the UI it's not sometimes, AFAIK the CLI doesn't have this problem
Using UI OCP-version-4.8.0 you can not connect at all to a VM running on CNV-version-2.6
workaround - use CLI, or upgrade to CNV-version-4.8.0

Comment (yaacov) [reply virt/virt-4-8-release-notes.adoc:248]: note: a fixed is planed for OCP-4.8.1, and then we could VNC into a VM running on CNV-2.6.z

**Review by yaacov — CHANGES_REQUESTED** [review]
In the UI, connection to VNC using OCP-4.8.0 to CNV-2.6.5 will always fail (not sometimes), a fix will be in OCP-4.8.1

File: virt/virt-4-8-release-notes.adoc (hunk 1 above)
Comment (mgarrellRH) [reply virt/virt-4-8-release-notes.adoc]: I've made changes based on the comments. What do you think?

**Review by yaacov — APPROVED** [review]

File: virt/virt-4-8-release-notes.adoc [hunk 2]
```diff
@@ -214,3 +214,38 @@ No model registered for VirtualMachines
...
++
+[source]
+----
+Error creating virtual machine
+----
++
+*** As a workaround, copy the manifest and create the virtual machine from the CLI. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1979116[*BZ#1979116*])
++
+// fix targeted for 4.8.1
+** When connecting to the VNC console by using the {VirtProductName} web console and `virtctl` command, the VNC console always fails to respond. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1977037[*BZ#1977037*])
```
Comment (gouyang) [inline virt/virt-4-8-release-notes.adoc:249]: By looking at other places, seems no need to add a bug link here.

File: virt/virt-4-8-release-notes.adoc [hunk 3]
```diff
@@ -214,3 +214,38 @@ No model registered for VirtualMachines
...
++
+*** As a workaround, create the virtual machine from the CLI or upgrade to CNV 4.8. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1979116[*BZ#1979116*])
```
Comment (gouyang) [inline virt/virt-4-8-release-notes.adoc:251]: The bug number should be BZ#1977037, isn't it?

Comment (mgarrellRH) [reply virt/virt-4-8-release-notes.adoc]: Yes, you are correct. I made a copy/paste error that included the 1977037 number. It's been removed.

File: virt/virt-4-8-release-notes.adoc (hunk 2 above)
Comment (mgarrellRH) [reply virt/virt-4-8-release-notes.adoc:249]: Removed

**Comment by gouyang** [comment]
LGTM

File: virt/virt-4-8-release-notes.adoc (hunk 2 above)
Comment (ousleyp) [reply virt/virt-4-8-release-notes.adoc:249]: I just looked at this bug and it appears that the `virtctl` command does not have this issue (comments 6 and 11). (The user didn't have the correct package installed.) Please reword this to specify that this only happens in the web console.

**Review by ousleyp — COMMENTED** [review]
Some things to address; please let me know if you have any questions about my comments. Thanks, Matthew!

**Review by ousleyp — APPROVED** [review]
